src/data/users.tsx

```diff
@@ -966,6 +966,15 @@ const Users: User[] = [
     author: 'https://bsky.app/profile/did:plc:ubz4oedvsb7dsuncqi5jb7o2',
     tags: ['bot', 'opensource'],
   },
+    {
+    title: 'Flight Emergency Alerts',
+    description: 'Bot alerting of aircraft indicating emergencies',
+    preview: require('./showcase/example-1.png'),
+    website: 'https://bsky.app/profile/did:plc:at2e7q6kwfhdwv7267mjsflz',
```

Comment:
I think you might have accidentally put the link to your profile in the `website` field instead of the link to the bot?